apps/api/lib/api_web/router.ex

```diff
@@ -10,5 +10,8 @@ defmodule ApiWeb.Router do
 
     resources "/users", UsersController, only: [:index, :show, :create, :update, :delete]
     resources "/posts", PostsController, only: [:index, :create]
+    get "/events", EventsController, :index
+    get "/events/:id", EventsController, :show
+    get "/events/operation/:operation", EventsController, :show
```

Comment:
I think this would work better as a parameter to `/events` instead of a separate route.

Reply:
I agree. It should be a query param for the `index` action. Then, you could define routes as:

```elixir
resources "/events", EventsController, only: [:index, :show]
```

Reply:
Actually, I thought about something like `/events?operation=user_created`. In other words, handling `def index(conn, %{"operation" => operation})` in the controller without any extra endpoint in routing.

The reason is that these actions are exactly the same except for an extra filter. It is the more conventional approach in building API (for example http://jsonapi.org/format/#fetching-filtering).